### ISSUE TITLE
hematospermia

### ISSUE THREAD

**Issue by nklsbckmnn** [obophenotype/human-phenotype-ontology#11061]
**Preferred term label:**
hematospermia

**Synonyms**
hemospermia

**Definition (free text, please give PubMed ID)**
The presence of blood in the seminal fluid.

**Parent term (use hpo.jax.org/app)**
Abnormal male reproductive system physiology (HP:0012874)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0009-0009-6791-365X

**Comment by pnrobinson** [comment]
@nklsbckmnn 
Adding new term: 
- ID: HP:6001268
- label: Hematospermia
- parents: [Abnormal male reproductive system physiology (HP:0012874)](https://hpo.jax.org/app/browse/term/HP:0012874)
- pmid: 
- def: The presence of blood in the seminal fluid.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11061
- synonyms: Hemospermia(NONE)
